composer.json

```diff
@@ -58,11 +58,11 @@
         "prestashop/ps_customtext": "^4",
         "prestashop/ps_emailsubscription": "^2",
         "prestashop/ps_facetedsearch": "^2",
-        "prestashop/ps_faviconnotificationbo": "^1",
+        "prestashop/ps_faviconnotificationbo": "^2",
         "prestashop/ps_featuredproducts": "^2",
         "prestashop/ps_imageslider": "^3",
         "prestashop/ps_languageselector": "^2",
-        "prestashop/ps_linklist": "^2",
+        "prestashop/ps_linklist": "^3",
```

Comment:
Looks like we cannot accept that : 

```
Warning: Uncaught Symfony\Component\Debug\Exception\ContextErrorException: Warning: require_once(/home/travis/build/PrestaShop/PrestaShop/modules/ps_linklist/vendor/autoload.php): failed to open stream: No such file or directory in /home/travis/build/PrestaShop/PrestaShop/modules/ps_linklist/ps_linklist.php:30
```

https://travis-ci.org/PrestaShop/PrestaShop/jobs/530335151#L632

Reply:
It should work now 🤞